src/Common/Core/Test/Fakes/Shell/TestServiceManager.cs

```diff
@@ -17,7 +17,7 @@ public class TestServiceManager: ServiceManager {
         public override T GetService<T>(Type type = null) {
             // First try internal services
             var service = base.GetService<T>(type);
-            return service ?? _exportProvider.GetExportedValueOrDefault<T>();
+            return service ?? _exportProvider?.GetExportedValueOrDefault<T>();
```

Comment:
Theoretically yes. It is a test implementation so it is possible that test decides not to use MEF and just supply few basic services.